### PR TITLE
Use cells3d image in README example

### LINKED ISSUE #3085: Bringing back the example from the old napari homepage
## 📚 Documentation

Hi @sofroniewn @kne42, 

I was at a conference this weekend and met a postdoc doing work on tumors at a Childrens Hospital. I wanted to show her a quick demo of napari that used to be on the landing page. The GIF/image of the cell was the one that I was thinking about. I eventually found something on the CZI website. 

I love the docs. Awesome job all around. 

If you point me in the direction of the old homepage, I would be happy to make a PR to the docs. :smile:

### PR DESCRIPTION
**Pull request by GenevieveBuckley**
# Description
There's been a request for napari to use more biological images in the documentation (instead of the current scikit-image astronaut image). This PR updates the README example to use the `cells3d` demo dataset from scikit-image.

## Type of change
- [x] This change is a documentation update

# References
Closes https://github.com/napari/napari/issues/3085

# How has this been tested?
No tests needed, documentation change only.

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~
- [ ] ~~If I included new strings, I have used `trans.` to make them localizable.~~

### CODE REVIEW

**Comment by GenevieveBuckley** [comment]
Note: I thought about whether we should use jupytext/myst for the example. 
I didn't do that with this PR because:
1. When people see the napari repo at github.com/napari/napari we still want them to see the images rendered, at least for the README. I don't think that will happen if I introduce jupytext here (correct me if I'm wrong though)
2. The `screenshot-add-image.png` file is used in a second location, `index.md`, so we shouldn't get rid of it just yet.

**Comment by tlambert03** [comment]
Yep!  Readme needs to remain static.  Thanks @GenevieveBuckley!

**Review by tlambert03 — APPROVED** [review]

**Comment by Carreau** [comment]
I'm confused why  resources/screenshot-add-image.png has been removed? Is the plan to re-add it with the new cells3d ?  

[edit] If that's the case docs/images/screenshot-add-image.png likely need update as well, or better, nuke the `resources` folder and make current references that point to resources/screenshot-add-image.png  point to docs/images/screenshot-add-image.png ?[/edit]

File: README.md [hunk 1]
```diff
@@ -106,7 +106,11 @@ From inside an IPython shell, you can open up an interactive viewer by calling
 from skimage import data
 import napari
 
-viewer = napari.view_image(data.astronaut(), rgb=True)
+viewer = napari.Viewer()
+viewer.open_sample('scikit-image', 'cells3d')
+# viewer.open_image('my/image/filename.zarr')  # or look at your own data
+
+viewer.dims.ndisplay = 3  # selects 3D mode
```
Comment (jni) [inline README.md]: I'd propose the following, as I don't love highlighting `open_sample` over `view_image` in the readme:

```suggestion
viewer = napari.view_image(data.cells3d(), channel_axis=1, ndisplay=3)
```

File: README.md [hunk 2]
```diff
@@ -117,7 +121,8 @@ To use napari from inside a script, use `napari.run()`:
 from skimage import data
 import napari
 
-viewer = napari.view_image(data.astronaut(), rgb=True)
+viewer = napari.Viewer()
+viewer.open_sample('scikit-image', 'cells3d')
```
Comment (jni) [inline README.md]:
```suggestion
viewer = napari.view_image(data.cells3d(), channel_axis=1, ndisplay=3)
```

As above.

**Comment by jni** [comment]
I agree with @Carreau, the screenshot needs to be replaced. Perhaps we can host it online rather than in the repo though?

**Comment by GenevieveBuckley** [comment]
> I'm confused why resources/screenshot-add-image.png has been removed? Is the plan to re-add it with the new cells3d ?

I *did* replace the screenshot. 

I think you're both getting confused because there is a red piece of text next to the filename - it's red because the new screenshot is smaller and so the overall file size has gone down. You can see the new screenshot under the green "Added" heading to right hand side. Circled here:
Screen Shot 2022-01-19 at 4 59 35 pm

**Comment by grlee77** [comment]
> I did replace the screenshot.

Cool, I had not seen that GitHub feature for comparing the old/new images. It is pretty nice with the tabbed options at the bottom!

Looks like this just needs a conflict resolved and then can go in?

**Comment by GenevieveBuckley** [comment]
I think the reason for the merge conflict was the introduction of git-lfs. I think it's ok now.

**Comment by GenevieveBuckley** [comment]
Merging this now. 
It's been approved, the CI checks have passed, and git-lfs is enabled for the image in this PR. Looks good to go.

**Pull request merged**